Do-ItCore/Organization/NotificationManager.swift

```diff
@@ -0,0 +1,33 @@
+//
+//  NotificationManager.swift
+//  Do-ItCore
+//
+//  Created by Cesar F. Chacon on 2/27/19.
+//  Copyright © 2019 The Swifter Picker-Uppers. All rights reserved.
+//
+
+import UserNotifications
+
+class NotificationManager {
+    let center = UNUserNotificationCenter.current()
+
+    func requestAuthorization() {
+        center.requestAuthorization(options: [.alert, .sound]) {
+            (granted, error) in
+                // do something if granted is false, user did not allow notifications
+        }
+    }
+
+    func setTrigger(_ doIt: DoIt) {
+        let content = UNMutableNotificationContent()
+        content.title = "Your DoIt is due " + doIt.dueDate.description
+        content.body = "Go finish it"
```

Comment:
For body, use something like "Be sure to complete {do-it assignment name} within {time delta between notification delivery time and do-it due date}."